src/components/alert/alert.stories.js

```diff
@@ -0,0 +1,139 @@
+import { html } from "lit-html";
+import "../button";
+import "../alert";
+import styles from "./styles.scss";
+import { stylesheet } from "../../../.storybook/utils.js";
+
+export default {
+  title: "custom-elements/vl-alert",
+  decorators: [(story) => html`${stylesheet(styles)}${story()}`],
+  args: {
+    title: "Opgelet!",
+    icon: "warning",
+    size: "normal",
+    type: "info",
+    closable: false,
+  },
+  argTypes: {
+    title: {
+      name: "data-vl-title",
+      type: { summary: "string" },
+      description: "Attribuut wordt gebruikt om de titel te bepalen.",
+      table: {
+        defaultValue: { summary: "" },
+      },
+    },
+    icon: {
+      name: "data-vl-icon",
+      type: "select",
+      options: ["warning", "check", "info-circle"],
```

Comment:
Omdat elk icon vaak gepaard gaat met een bepaalde type, zou ik ook per type een story toevoegen. Zoals hier: https://webcomponenten.omgeving.vlaanderen.be/demo/vl-alert.html 

Een voor Error, Warning, Success en Info

Reply:
Done.